daprdocs/content/en/reference/components-reference/supported-secret-stores/kubernetes-secret-store.md

```diff
@@ -30,8 +30,11 @@ spec:
   type: secretstores.kubernetes
   version: v1
   metadata:
-  - name: ""
 ```
+
+## Spec metadata fields
+For the Kubernetes secret store component there are no additional metadata attributes that need to be set during setup.
```

Comment:
```suggestion
For the Kubernetes secret store component there are no metadata attributes.
```

Reply:
For me, just add a comma after "component".

Reply:
updated